jupyter_book/commands/__init__.py

```diff
@@ -117,106 +117,61 @@ def build(path_book, path_output, config, toc, warningiserror, builder):
-            path_output_rel = Path(op.relpath(OUTPUT_PATH, Path()))
-            path_index = path_output_rel.joinpath("index.html")
-            _message_box(
-                f"""\
-            Finished generating HTML for book.
-
-            Your book's HTML pages are here:
-                {path_output_rel}{os.sep}
-
-            You can look at your book by opening this file in a browser:
-                {path_index}
-
-            Or paste this line directly into your browser bar:
-                file://{path_index.resolve()}\
-            """
-            )
-        if builder == "linkcheck":
-            _message_box("All links in your book are valid. See above for details.")
-        if builder == "pdfhtml":
-            print("Finished generating HTML for book...")
-            print("Converting book HTML into PDF...")
-            path_pdf_output = OUTPUT_PATH.parent.joinpath("pdf")
-            path_pdf_output.mkdir(exist_ok=True)
-            path_pdf_output = path_pdf_output.joinpath("book.pdf")
-            html_to_pdf(OUTPUT_PATH.joinpath("index.html"), path_pdf_output)
-            path_pdf_output_rel = Path(op.relpath(path_pdf_output, Path()))
-            _message_box(
-                f"""\
-            Finished generating PDF via HTML for book. Your PDF is here:
-
-                {path_pdf_output_rel}\
-            """
-            )
-        if builder == "pdflatex":
-            print("Finished generating latex for book...")
-            print("Converting book latex into PDF...")
-            # Convert to PDF via tex and template built Makefile and make.bat
-            if sys.platform == "win32":
-                makecmd = os.environ.get("MAKE", "make.bat")
-            else:
-                makecmd = os.environ.get("MAKE", "make")
-            try:
-                with cd(OUTPUT_PATH):
-                    output = subprocess.run([makecmd, "all-pdf"])
-                    if output.returncode != 0:
-                        _error("Error: Failed to build pdf")
-                        return output.returncode
-                _message_box(
-                    f"""\
-                A PDF of your book can be found at:
-
-                    {OUTPUT_PATH}
-                """
-                )
-            except OSError:
-                _error("Error: Failed to run: %s" % makecmd)
-                return 1
+    builder_specific_actions(exc, builder, OUTPUT_PATH, "book")
 
 
 @main.command()
 @click.argument("path-page")
 @click.option("--path-output", default=None, help="Path to the output artifacts")
 @click.option("--config", default=None, help="Path to the YAML configuration file")
+@click.option("-W", "--warningiserror", is_flag=True, help="Error on warnings.")
 @click.option(
-    "--execute/--no-execute",
-    default=True,
-    help="Whether to execute the notebook. Default is --execute",
+    "--builder",
+    default="html",
+    help="Which builder to use. Must be one of {BUILDER_OPTIONS}",
 )
```

Comment:
where did the execute option go? Am I missing it?

Reply:
Because we have all the options for execute now, like `cache`, `force`, `off`  in the `_config` file, so it did not make sense to have on and off option here.